### ISSUE TITLE
A11y: Colour contrast

### ISSUE THREAD

**Issue by ashleypeacock** [melissaegg/trauma-informed-design-resources#1]
### Issue instance 1
On the home page, within the resource cards with the type "Article"; the pink background and white text in the heading , fails minimum colour contrast requirements.

<img width="753" alt="Screenshot 2024-07-24 at 01 27 22" src="https://github.com/user-attachments/assets/14e569ec-c8dc-4c1c-889b-9afb764adb02">

### Issue instance 2
On the home page, within the resource cards with the type "Tools and Toolkits"; the green background and white text in the heading. This fails minimum colour contrast requirements.

<img width="746" alt="Screenshot 2024-07-24 at 01 33 47" src="https://github.com/user-attachments/assets/6422ea8b-ae32-40a0-9f18-21e6d1a8d406">

### Recommendations 
* Update colour contrast to meet minimum guidlines
* Test using the [Tpig colour contrast checker ](https://www.tpgi.com/color-contrast-checker/)

**Comment by ashleypeacock** [comment]
There's a few of the navigation links that break colour contrast; i'd recommend using the colour contrast analyser to review.
